File: time.c
#include <stdio.h>


int hour; 


int main (){
    size_t now; 
    struct tm*now_tm; 
    
    now = time(NULL);
    now_tm = localtime(&now);
    hour = now_tm->tm_hour; 
    printf("%d", hour);  

    if (hour <= 12){
        printf("Good morning! :)\n");
    }else if (hour <= 18){
        printf("Good afternoon! :)\n");
    }else if (hour <= 20){
        printf("Good evening :)!\n");
    }else{
        printf("Good NIGHT!\n GO TO BED!");
    }

    return 0; 
}

File: time.py
import datetime

time = datetime.datetime.now().hour 
print(time) 
if time <= 12:
    print("Top of the day to ya!")

elif time <= 18:
    print("Afternoon to you!")

elif time <= 16:
    printf("Good evening!")

else:
    print("What the HECK are you doing up?!")